### PR TITLE
Capture view UI improvements.

### PR DESCRIPTION
**Pull request by pierricgimmig**
- Show thread names above thread track.
    - Get rid of thread "handle" on the left of thread tracks.
    - Expose TimeGraphLayout values in ImGui window (press 'i').
    - Tweaked TimeGraphLayout values.

### CODE REVIEW

**Comment by pierricgimmig** [comment]
OrbitUiTweaker

File: OrbitGl/TimeGraphLayout.cpp [hunk 1]
```diff
@@ -162,3 +167,38 @@ void TimeGraphLayout::SetSortedThreadIds(
...
+#define FLOAT_SLIDER(x)                     \
+  if (ImGui::SliderFloat(#x, &x, 0, 100)) { \
+    needs_redraw = true;                    \
+  }
+
+//-----------------------------------------------------------------------------
+#define FLOAT_SLIDER_MIN_MAX(x, min, max)     \
+  if (ImGui::SliderFloat(#x, &x, min, max)) { \
+    needs_redraw = true;                      \
+  }
```
Comment (dimitry-) [inline OrbitGl/TimeGraphLayout.cpp]: is there a reason for using macro instead of function?

File: OrbitGl/Track.cpp [hunk 2]
```diff
@@ -10,7 +10,7 @@
 #include "absl/strings/str_format.h"
 
 float TEXT_Z = -0.004f;
-float TRACK_Z = -0.005f;
+float TRACK_Z = -0.1f;
```
Comment (dimitry-) [inline OrbitGl/Track.cpp]: constexpr const float?

File: OrbitGl/TimeGraphLayout.cpp (hunk 1 above)
Comment (pierricgimmig) [reply OrbitGl/TimeGraphLayout.cpp]: To have access to the preprocessor's "stringify" functionality ("#x"), and to reduce a block of ugly boilerplate code to a short one liner debug utility.

```
FLOAT_SLIDER(m_SpaceBetweenTracksAndThread);
FLOAT_SLIDER(m_SpaceBetweenCoresAndThread);
```
vs
```
  if (ImGui::SliderFloat("m_SpaceBetweenTracksAndThread",
                         &m_SpaceBetweenTracksAndThread, 0, 100.f)) {
    needs_redraw = true;
  }
  if (ImGui::SliderFloat("m_SpaceBetweenCoresAndThread",
                         &m_SpaceBetweenCoresAndThread, 0, 100.f)) {
    needs_redraw = true;
  }
```

File: OrbitGl/Track.cpp (hunk 2 above)
Comment (pierricgimmig) [reply OrbitGl/Track.cpp]: It's handy to be able to tweak those at runtime.  I moved them to TimeGraphLayout and exposed them through the ImGui debug UI.

**Pull request merged**